File: morkato-api/src/main/java/org/morkato/api/exception/guild/GuildNotFoundError.java
package org.morkato.api.exception.guild;

import lombok.Getter;
import org.morkato.api.exception.NotFoundError;

@Getter
public class GuildNotFoundError extends NotFoundError{
  private final String id;
  public GuildNotFoundError(String id) {
    super("Guild with id: " + id + " is not found.");
    this.id = id;
  }
}

File: morkato-http-api/src/main/kotlin/org/morkato/http/serv/exception/model/GuildNotFoundError.kt
package org.morkato.http.serv.exception.model

import org.morkato.http.serv.exception.ModelType
import org.morkato.http.serv.exception.NotFoundError

class GuildNotFoundError(extra: Map<String, Any?>) : NotFoundError(ModelType.GUILD, extra) {
}